ArcadiaParties.Data.Abstractions/DTOs/UserDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ArcadiaParties.Data.Abstractions.DTOs
+{
+    public class UserDTO
+    {
+        public string Identity { get; set; }
+
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public DateTime BirthDate { get; set; }
+
+        public DateTime HireDate { get; set; }
+
+        public ICollection<string> UserRoles { get; set; }
```

Comment:
`ICollection` variable implies possibility to add or remove elements. You don't need it now, so the type should be changed. You can use `IReadOnlyCollection` or even `IEnumerable`, because it is enough for now.

Reply:
Collection type have been changed